modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a custom ingress controller file.
+
+.. Create a file that defines the custom ingress controller custom resource (CR):
+
++
+.Example `custom-ingress-controller.yaml` file
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+    name: custom<1>
+    namespace: openshift-ingress-operator
+spec:
+defaultCertificate: <2>
+    name: custom-ingress-custom-certs
+domain: "custom-domain.mycompany.com"<3>
```

Comment:
```suggestion
domain: <"custom_domain.mycompany.com"> <3>
```

Reply:
User replaced values should use an underscore: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

Reply:
I would also suggest adding a space between the callout and the code.

Reply:
Yeah I like the space better